Gemfile

```diff
@@ -16,3 +16,5 @@ gemspec
 
 # To use a debugger
 # gem 'byebug', group: [:development, :test]
+
+gem "erb_lint", "~> 0.9.0"
```

Comment:
Just wanted to check if `erb_lint` should/could be included in the gemspec file as a development dependency - https://github.com/alphagov/govuk_publishing_components/blob/main/govuk_publishing_components.gemspec#L29-L46

Reply:
Thanks @MartinJJones - have added this now 👍